docs/build-the-builder.md

```diff
@@ -0,0 +1,185 @@
+ - prepare the local environment to build the builder container
+ - how to update the rpm dependencies with bazeldnf
+ - build the builder container
+ - push the builder image to a custom registry
+ - invoke the new test builder image to build new kubevirt operator images
+ - push the new kubevirt operator images to a custom registry
+
+
+## Prerequisites
+
+### Podman or Docker
+
+install podman or docker
+
+On Ubuntu,
+
+```
+$ apt get install -y podman
+```
+
+or 
+
+```
+$ apt get install -y docker
+```
+
+On RHEL or CENTOS
+
+```
+$ dnf install -y podman
+```
+
+### Configure your read/write access to a container registry
+
+You will first need to create read and write access to an appropriate namespace in a docker registry
+
+[`docker.io`](https://docs.docker.com/docker-id/) or [`quay.io`](https://docs.quay.io/solution/getting-started.html) or [`icr.io`](https://cloud.ibm.com/docs/Registry?topic=Registry-getting-started) are popular registries to use for this purpose.  Then set up a repository or namespace to hold the container you are building, e.g. `builder` or, where the namespace needs to be more unique, `kubevirt-builder`. 
+
+##  Install bazel and dependencies in order to containerize the builder 
+
+Bazel binary distribution images for version 5.3.0 for the Arm and x86 architectures are available from https://github.com/bazelbuild/bazel/releases/tag/5.3.0
+
+This, the dependent libraries and build script configuration will constitute the containerized KubeVirt build toolchain.  For architectures other than arm and x86, e.g. IBM ppc64le and s390x, you need to build bazel from scratch on your build-the-builder box. 
+
+Currently, [a scripted build for bazel is available for Ubuntu on s390x](https://github.com/linux-on-ibm-z/docs/wiki/Building-TensorFlow) step 1.2 'Build Bazel v5.3.0'.
+
+The Dockerfile you'll be building with that bazel installation expects to 
+find the bazel exectable in `kubevirt/hack/builder/bazel`, so copy it there, e.g.:
+
+```
+$ cp /usr/local/bin/bazel ~/kubevirt/hack/builder/bazel
+```
+
+## Build and publish the Bazel builder container
+
+### Set environment variables for building the builder -- all architectures, all registries
+
+Note that the default system architecture is amd64, so if you do not set ARCH as an environment variable, you will build a docker container with x86 object format binaries in it. 
+
+```
+export ARCH="<your-system-architecture>"
+export DOCKER_PREFIX="<your-registry-URL>/<your-kubevirt-namespace>" 
+export DOCKER_IMAGE="builder" 
+export VERSION="<your-docker-tag>"
+```
+where <your-system-architecture> is either `amd64` or `arm64` or `s390x` or `ppc64le` for x86, ARM, s390x or Power systems, respectively.
+
+
+`make builder-build` invokes the build of the bazel builder container. 
+
+This tags your image with namespace you set up in your environment variables.
+
+Edit the file `hack/dockerized`, and update the parameter `KUBEVIRT_BUILDER_IMAGE` to reflect this tag. 
```

Comment:
I see that `KUBEVIRT_BUILDER_IMAGE` is also accepted if an env var is provided.
So I think you do not need to edit the file.

Reply:
This procedure was tested on an s390x LPAR running Ubuntu, using the IBM cloud container registry.